ios/CoopCycle.xcodeproj/project.pbxproj

```diff
@@ -135,55 +101,44 @@
 		13B07FAE1A68108700A75B9A /* CoopCycle */ = {
 			isa = PBXGroup;
 			children = (
-				FA53F9442047D093004395DF /* CoopCycle.entitlements */,
-				008F07F21AC5B25A0029DE68 /* main.jsbundle */,
 				13B07FAF1A68108700A75B9A /* AppDelegate.h */,
 				13B07FB01A68108700A75B9A /* AppDelegate.mm */,
 				13B07FB51A68108700A75B9A /* Images.xcassets */,
 				13B07FB61A68108700A75B9A /* Info.plist */,
+				81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */,
 				13B07FB71A68108700A75B9A /* main.m */,
-				84E9E2D624E3EA5B00740A6D /* Infoplist.strings */,
-				8435002D2BB3E1AF0013D5F8 /* LaunchScreen.storyboard */,
+				13B07FB81A68108700A75B9A /* PrivacyInfo.xcprivacy */,
 			);
 			name = CoopCycle;
 			sourceTree = "<group>";
 		};
-		16D169907EB9F70DE0842958 /* ExpoModulesProviders */ = {
+		2D16E6871FA4F8E400B85C8A /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
-				9A033E64AFF479081FA0F18E /* CoopCycle */,
-				92279F0093D1CBA15C7AF5C2 /* CoopCycleTests */,
+				ED297162215061F000B7C4FE /* JavaScriptCore.framework */,
+				F108F404317D382A7A85783E /* Pods_CoopCycle.framework */,
+				71CC223F28428DA396CC6344 /* Pods_CoopCycle_CoopCycleTests.framework */,
 			);
-			name = ExpoModulesProviders;
+			name = Frameworks;
 			sourceTree = "<group>";
 		};
-		2D16E6871FA4F8E400B85C8A /* Frameworks */ = {
+		832341AE1AAA6A7D00B99B32 /* Libraries */ = {
 			isa = PBXGroup;
 			children = (
-				ED297162215061F000B7C4FE /* JavaScriptCore.framework */,
-				2D16E6891FA4F8E400B85C8A /* libReact.a */,
-				F8A8B7590270EE06D2D26777 /* Pods_CoopCycle.framework */,
-				A5763258E52216DBA45B1C3D /* Pods_CoopCycle_CoopCycleTests.framework */,
 			);
-			name = Frameworks;
+			name = Libraries;
 			sourceTree = "<group>";
 		};
 		83CBB9F61A601CBA00E9B192 = {
 			isa = PBXGroup;
 			children = (
-				845246CE2344832500A3A1B2 /* GoogleService-Info.plist */,
```

Comment:
This causes an error: 

> Could not get GOOGLE_APP_ID in Google Services file from build environment

Not sure why/how this was removed, but it breaks the build